File: csharp/senhaforte.cs
using System;

class Program {
    static void Main() {
        string senha = "adm";
        bool ver = (senha.Contains("@") || senha.Contains("#") || senha.Contains("$") || senha.Contains("%") || senha.Contains("^") || senha.Contains("&") || senha.Contains("*") || senha.Contains("_")) && senha.Length >= 6;
        Console.WriteLine(ver);
    }
}

File: javascript/senhaforte.js


let senha = "adm";

let ver = (senha.includes("@") || senha.includes("#")|| senha.includes("$") || senha.includes("%") || senha.includes("^") || senha.includes("&")
|| senha.includes("*") || senha.includes("_")) && (senha.length>=6);

console.log(ver);